demos/admin_panel_bot/screens.py

```diff
@@ -0,0 +1,105 @@
+"""The module contains the screens the bot consists of."""
+
+from hammett.conf import settings
+from hammett.core import Button, Screen
+from hammett.core.constants import SourcesTypes
+from hammett.core.handlers import register_button_handler
+from hammett.core.hider import ONLY_FOR_ADMIN, Hider
+from hammett.core.mixins import StartMixin
+
+ADMIN_PANEL_SCREEN_DESCRIPTION = (
+    "This is <b>AdminPanelScreen</b>. You've managed to move here, "
+    "because you could see the button that moves to this screen.\n"
+    "\n"
+    "Now remove yourself from the admin group to see what will change."
+)
+
+MAIN_MENU_SCREEN_DESCRIPTION = (
+    "Welcome to HammettAdminPanelBot!\n"
+    "\n"
+    "This is <b>MainMenuScreen</b>, which actually contains <b>two</b> buttons. "
+    "But only <b>one</b> of them is visible for the users who doesn't have the <i>admin</i> role."
```

Comment:
`This is the <b>Main Menu</b> screen, which actually contains <b>two</b> buttons. However, only <b>one</b> of them is visible to users who are not in the <i>admin</i> group.`